enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/CompositeNodeKeyConstraintAcceptanceTest.scala

```diff
@@ -31,36 +29,41 @@ import org.neo4j.test.TestEnterpriseGraphDatabaseFactory
 import scala.collection.JavaConverters._
 import scala.collection.Map
 
-class CompositeNodeKeyConstraintAcceptanceTest extends ExecutionEngineFunSuite with NewPlannerTestSupport {
+class CompositeNodeKeyConstraintAcceptanceTest extends ExecutionEngineFunSuite with CypherComparisonSupport {
 
   override protected def createGraphDatabase(config: Map[Setting[_], String] = databaseConfig()): GraphDatabaseCypherService = {
     new GraphDatabaseCypherService(new TestEnterpriseGraphDatabaseFactory().newImpermanentDatabase(config.asJava))
   }
 
   test("should be able to create and remove single property NODE KEY") {
     // When
-    exec("CREATE CONSTRAINT ON (n:Person) ASSERT (n.email) IS NODE KEY")
+    succeedWith(Configs.Procs,
```

Comment:
This line is really just a test setup command, not a test itself. However, written like this it is currently only testing that `CREATE CONSTRAINT` commands are disallowed with any runtime or planner or version prefixes, which is certainly out of scope for this test. So we end up running this schema command on all scenarios where all scehma commands will fail, and expect failure. This is very redundant. Let's rather have this test only test what it was meant to test.